### ISSUE TITLE
We will probably eventually make Save/Load publicly available through our normal licensing

### ISSUE THREAD

**Issue by samreid** [phetsims/together#25]
@kathy-phet said for now it is ok to develop Save/Load under together.js but that in the future (say >1 years from now) we may want to make Save/Load available in all of our public (non-enhanced) sims.  Once we decide on a user interface for save/load (including whether a state can be loaded via query parameters), it will be easier to understand how its implementation will relate to the other together.js features, such as customizability via query parameters.

**Comment by pixelzoom** [comment]
Duplicate of https://github.com/phetsims/joist/issues/103?

**Comment by samreid** [comment]
Yes, it is the same issue.  Not sure where the best place for an issue is.  Perhaps in Joist?  But I still want it to be on the radar here in Together.

**Comment by samreid** [comment]
Whether we do it now or later, untangling save/load from the rest of together.js will be a big mess.  Not sure if we can circumvent a technical complication via changes to our license for together.js.

**Comment by samreid** [comment]
At today's meeting, @kathy-phet said she would like to wait a year or so to see how this suite of features is received by our users, before splitting things up.

**Comment by kathy-phet** [comment]
Changed title of this issue - hope that doesn't mess anyone up.

**Comment by samreid** [comment]
Thanks @kathy-phet, the change looks good.

**Comment by pixelzoom** [comment]
As mentioned above, related to (or duplicate of?) https://github.com/phetsims/joist/issues/103.  Support for save/load, and generalization of that feature, has already come up in the context of the "basic" version of sims.
